.github/workflows/main.yml

```diff
@@ -0,0 +1,156 @@
+---
+
+name: main
```

Comment:
OTOH, I'm not fully convinced with `main` for the `ci` specific workflows. Somehow it looks like it refers to the `main` branch. What do you think by using `test` or `ci`? I've been using `ci` in the migrations I've been doing but if I recall @reakaleek used `test` in some other places.

What are your thoughts?

Reply:
I've used `test` in the beginning but slowly started to prefer `ci`. The main reason was the status check being named e.g `test / test`, whereas `ci / test` looked more pleasant.

Reply:
I renamed the workflow to `ci`.